.github/workflows/run_periodic_tests.yml

```
@@ -38,42 +38,58 @@ jobs:
           python -m pip install pre-commit
           pre-commit run -a
 
-  build:
+  run_unit_tests:
     needs: style
```

```suggestion
  run_unit_tests:
```

Could you remove the `style` job above as well? The reason is that we would not want the style checks to run for a nightly job, because the tests won't run if they fail